backend/database/migrations/2026_06_16_120000_backfill_ready_batch_step_status.php

```
@@ -0,0 +1,40 @@
+ * DONE/SKIPPED) to READY so in-flight batches match the new model. No-op for
+ * finished batches (they have no PENDING steps).
+ */
+return new class extends Migration
+{
+    public function up(): void
+    {
+        if (! Schema::hasTable('batches') || ! Schema::hasTable('batch_steps')) {
+            return;
+        }
+
+        Batch::query()->with('steps')->chunkById(200, function ($batches) {
```

PHPStan (2.2.2)

[error] Call to an undefined static method App\Models\Batch::query().


(staticMethod.notFound)